Casks/the-battle-for-wesnoth.rb

```diff
@@ -10,7 +13,7 @@
 
   livecheck do
     url :homepage
-    regex(/href=.*?Wesnoth[._-]v?(\d+(?:\.\d+)+[a-z]?)\.dmg/i)
+    regex(/href=.*?Wesnoth[._-]v?(\d+(?:\.\d*[02468])+(?:\.\d+)*[a-z]?)\.dmg/i)
```

Comment:
```suggestion
    regex(/href=.*?Wesnoth[._-]v?(\d+\.\d*[02468](?:\.\d+)*[a-z]?)\.dmg/i)
```

The part that matches the even-numbered minor version doesn't need to repeat, as the following `(?:\.\d+)*` is intended to match any numeric parts after the minor. If we address that issue, this regex will be in line with existing regexes where we match only versions with an even-numbered minor.